CHANGELOG.md

```diff
@@ -18,6 +19,47 @@ CHANGELOG
 
 </details>
 
+Swift 4.1
+---------
+
+* [SE-0185][]
+
+  Structs and enums now automatically receive synthesized implementations of `Equatable` and/or `Hashable` if all of their stored members conform to those protocols. You must explicitly declare that the type conforms to `Equatable`/`Hashable` (not in an extension), but the compiler will provide the implementation of `==`/`hashValue` for you if you do not write one yourself.
```

Comment:
Nitpick: until the second sentence this suggests that the conformance is added automatically. Maybe a better way to phrase it would be "Structs and enums declared a conformance to `Equatable`/`Hashable` now get an automatically synthesized implementation of `==`/`hashValue`. For structs, all stored properties must be `Equatable`/`Hashable`. For enums, all enum cases with associated values must be `Equatable`/`Hashable`".

Reply:
Done.